### PR TITLE
Concept/date picker

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Screenshot 2021-09-21 at 12 27 35

Alignment issues

File: components/HackneyDS/FormGroup/index.js [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+  const outerClassName = className ? ` ${className}` : '';
   const nodeList = Array.isArray(children) ? children : [children];
   const errorClassList = error ? ' govuk-form-group--error' : '';
 
   return (
-    <div className={`govuk-form-group lbh-form-group${errorClassList}`}>
+    <div className={`govuk-form-group lbh-form-group${errorClassList}${outerClassName}`}>
       <fieldset className="govuk-fieldset lbh-fieldset">
-        <legend className="govuk-fieldset__legend">{title}</legend>
+        <legend className="govuk-fieldset__legend">{label}</legend>
```
Comment (akeamwoodsND) [inline components/HackneyDS/FormGroup/index.js:12]: - This change affects the RadioGroup storybook (rename prop in storybook to label).
- Change <legend> to <Label> component

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Each section should be two digits long (e.g. 2/2/12 -> 02/02/12)
Screenshot 2021-09-28 at 10 05 32

**Comment by mugnivenko** [comment]
(Fixed) Each section should be two digits long (e.g. 2/2/12 -> 02/02/12)

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Label prop missing on radio group storybook examples
Screenshot 2021-09-28 at 13 19 35

**Comment by mugnivenko** [comment]
(Done) Change prop (title) to (label)

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
`cursor: pointer;` over calendar icon

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Label is wrong on date picker, look at difference between date picker label and input label
Screenshot 2021-09-28 at 14 04 52
Screenshot 2021-09-28 at 14 05 05

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Why are there two sets of labels and hints?
Screenshot 2021-09-28 at 16 47 27

**Pull request merged**